### PR TITLE
Fix/dex 126 fix form interactions

### PR DESCRIPTION
**Pull request by mvashchuk-bc**
Fix form mechanic

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts [hunk 1]
```diff
@@ -10,6 +10,9 @@ import type { DexSwapForm } from './types'
 
 const { DEX_SWAP_FORM } = model.components.dex
 
+const getValidAmount = (amount: number | undefined) =>
```
Comment (dkremniov-bc) [inline packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts:13]: Minor: can be extracted to utils and unit tested there.

Comment (mvashchuk-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts]: Done

Comment (dkremniov-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts]: Would be nice to test it too. Maybe in the next PR if we are in the rush.

Comment (mvashchuk-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/dex/sagas.ts]: Yep, I'll start working on tests soon. Created a todo for me in order not to forget this one

**Pull request merged**